lib/vmdb/loggers.rb

```diff
@@ -96,6 +96,28 @@ def self.apply_config_value(config, logger, key)
         logger.level = new_level
       end
     end
+
+    def self.contents(log, last = 1000)
+      log = log.instance_variable_get(:@logdev)&.filename.to_s if log.kind_of?(Logger)
+      return "" unless File.file?(log)
```

Comment:
Not for this PR or even in the future if this is going away but I'm not sure why we're checking `file?`.  It should be checking `readable?`.